airbyte-integrations/connectors/source-goldcast-api/metadata.yaml

```
@@ -0,0 +1,35 @@
+data:
+  allowedHosts:
+    hosts:
+      - TODO # Please change to the hostname of the source.
+  registries:
+    oss:
+      enabled: true
+    cloud:
+      enabled: false
+  remoteRegistries:
+    pypi:
+      enabled: true
+      packageName: airbyte-source-goldcast-api
+  connectorBuildOptions:
+    # Please update to the latest version of the connector base image.
+    # https://hub.docker.com/r/airbyte/python-connector-base
+    # Please use the full address with sha256 hash to guarantee build reproducibility.
+    baseImage: docker.io/airbyte/python-connector-base:1.2.0@sha256:c22a9d97464b69d6ef01898edf3f8612dc11614f05a84984451dde195f337db9
+  connectorSubtype: api
+  connectorType: source
+  definitionId: c2c25d04-9bb1-4171-8a7a-bb7cead8add0
+  dockerImageTag: 0.1.0
+  dockerRepository: airbyte/source-goldcast-api
+  githubIssueLabel: source-goldcast-api
+  icon: goldcast-api.svg
+  license: MIT
+  name: Goldcast Api
```

```suggestion
  name: Goldcast
```

It would be rather obvious that it's an API ;)